### PR TITLE
string formatting done (try fixing all syntax errors)

### PR DESCRIPTION
**Pull request by yashagg2001**
Trying to solve the issue (old string formatting replaced by new f-string formatting) once again.

### CODE REVIEW

**Comment by andrewfullard** [comment]
You have left in merge lines `=======` etc. This is also incorrect syntax.

**Comment by andrewfullard** [comment]
Please follow the code quality guidelines at https://tardis-sn.github.io/tardis/development/code_quality.html so that the checks all pass

File: docs/conf.py [hunk 1]
```diff
@@ -100,7 +100,7 @@
 # This does not *have* to match the package name, but typically does
 project = u'TARDIS'
 author = u'TARDIS collaboration'
-copyright = '{0}-{1}, {2}'.format(2013, datetime.datetime.now().year, author)
+copyright = f'{2013}-{datetime.datetime.now().year}, {author}'
```
[the remark this answers is not in the document]
Comment (andrewfullard) [reply docs/conf.py:103]: That is correct. However, you have left tempCodeRunner.py files in your pull request which should not be part of the PR.

[the remark this answers is not in the document]
Comment (yashagg2001) [reply docs/conf.py:103]: okay, I will delete those files..

**Comment by andrewfullard** [comment]
Please resolve the conflicts in  
docs/conf.py
tardis/widgets/shell_info.py

**Comment by yashagg2001** [comment]
I am unable to see any conflicts in these files. Please tell me how to resolve it.

**Comment by yashagg2001** [comment]
I am unable to see any conflicts in these files. Please tell me how to resolve it.

**Review by andrewfullard — APPROVED** [review]

**Pull request merged**